.github/workflows/benchmark_execution.yml

```diff
@@ -105,6 +105,7 @@ jobs:
   run_benchmarks:
     needs: [generate_matrix]
     if: needs.generate_matrix.outputs.benchmark-matrix != '[]'
+    timeout-minutes: 1440
```

Comment:
👀 that's a long timeout

![image](https://github.com/openxla/iree/assets/4010439/fca8ce02-2021-40e9-84a5-c7cbf674064b)

I see that this PR is marked as a draft, but I'm curious - what's the rough plan for benchmarking these larger programs? Is < 5 minutes per benchmark configuration achievable on some/all platforms? Would we run these benchmarks on desktop and not mobile?

Reply:
I was running these to check how long it would take to benchmark on mobile. We won't be benchmarking regularly on mobile until we have the quantized versions running at a reasonable latency. For now, we'll need to stick to x86.